content/tutorials/noetic_quickstart.md

```diff
@@ -0,0 +1,140 @@
+recent version, or update it if you already have it downloaded.
+
+Open Foxglove Studio, and click the "Layouts" button on the left panel (second from top) and then click
+`Import Layout` button pictured below.
+{{< figure src="/tutorials/noetic_quickstart/foxglove_layout.png" width="400" >}} 
+
+Import the preset layout at:
+{{< highlight bash >}} mushr/mushr_utils/foxglove/foxglove_layout.json {{< / highlight >}}    
+
+Three panels should appear after selecting this layout, as pictured below. The left panel is for the datasource, the central panel is for the map, and the right panel is for teleop driving.
+
+{{< figure src="/tutorials/noetic_quickstart/layout_example.png" width="700" >}}                      
+
+The layout can be edited with the `Add Panel` button on the left sidebar if desired.
+
+## Connecting to Data With Foxglove Studio
+Now that Foxglove is set up, we can connect the visualization to our docker container ROS setup. Start the docker container:
+{{< highlight bash >}} $ mushr_noetic {{< / highlight >}}
+Once the `root` prompt appears, source the `.bashrc`. Note that this does not occur automatically in the docker container.
+
+{{< highlight bash >}} $ source .bashrc {{< / highlight >}}
+
+In the same terminal, start up the simulator with the command:
+
+{{< highlight bash >}} $ roslaunch mushr_sim teleop.launch {{< / highlight >}}
+
+After starting up, the simulator should print out a line similar to 
+
+{{< highlight bash >}} Rosbridge WebSocket server started at ws://0.0.0.0:9090 {{< / highlight >}}
+
+In foxglove, click the top button in the sidebar, labeled `Data Source`. Then select the Plus button in the left panel. This should open up an interface to connect to data. 
+
+{{< figure src="/tutorials/noetic_quickstart/data_panel.png" width="400" >}}
+
+Click the `Open Connection` button. Select `Rosbridge (ROS 1 & ROS 2)` as shown below.
+
+{{< figure src="/tutorials/noetic_quickstart/rosbridge.png" width="700" >}}
+
+Fill out the WebSocket URL with the url and port that the simulator output before, as shown in the image. The URL may differ from pictured. Then, click `Open` in the bottom right corner. The left sidebar should show that a connection has been made. The map should update and the car pose arrow should appear in the map panel in the center. The car can be driven using the right teleop panel.
+
+{{< figure src="/tutorials/noetic_quickstart/example.png" width="900" >}}
+
+TODO: Get robot model to appear & update image
+
+### Changing the map
+The map can be changed by editing the following file:
+{{< highlight bash >}} src/mushr_sim/launch/teleop.launch {{< / highlight >}}
+
+In this file, edit the line
+{{< highlight bash >}} <arg name="map" default="$(find mushr_sim)/maps/sandbox.yaml"/>{{< / highlight >}}
+to specify a different map. MuSHR provides some basic maps, but if you want to map your own space, try the [SLAM Tutorial for MuSHR](/tutorials/mapping).
+
```

Comment:
How can we teleport the car? Like `2D pose estimate` in rviz

Reply:
Yeah, they implemented this pretty recently in [this PR](https://github.com/foxglove/studio/pull/2719). Here's how to do it. Looking at the PR, I think that you can rotate the orientation of the point but I haven't figured out how to do that yet. I'll take another look later this week.

https://user-images.githubusercontent.com/45296521/163849786-1c47c905-f776-493f-99c3-3dfc6d4d28b1.mov

Reply:
I'll add a section for this too, thanks for the feedback!